apps/preview/next/pages/showcases/Gallery/GalleryHorizontalArrowKeyNavigation.tsx

```diff
@@ -11,35 +11,23 @@ import {
   SfIconChevronRight,
   type SfScrollableOnDragEndData,
 } from '@storefront-ui/react';
-import * as Images from './images';
 import classNames from 'classnames';
 
-const images = [
-  { imageSrc: Images.gallery1Src, alt: 'backpack1' },
-  { imageSrc: Images.gallery2Src, alt: 'backpack2' },
-  { imageSrc: Images.gallery3Src, alt: 'backpack3' },
-  { imageSrc: Images.gallery4Src, alt: 'backpack4' },
-  { imageSrc: Images.gallery5Src, alt: 'backpack5' },
-  { imageSrc: Images.gallery6Src, alt: 'backpack6' },
-  { imageSrc: Images.gallery7Src, alt: 'backpack7' },
-  { imageSrc: Images.gallery8Src, alt: 'backpack8' },
-  { imageSrc: Images.gallery9Src, alt: 'backpack9' },
-  { imageSrc: Images.gallery10Src, alt: 'backpack10' },
-  { imageSrc: Images.gallery11Src, alt: 'backpack11' },
-];
+const withBase = (filepath: string) =>
+  `http://localhost:3100/@assets/gallery/${filepath}`;
```

Comment:
It could be, and IMO it should be a shared util, so we could use it in all places. It'd let us easily change the base URL in the future. Could you extract this helper function?

Reply:
After brief chat with @Szymon-dziewonski I see we can't do this right now due to suggested solution. If we want to replace the sting on each environment, then we must keep the util on a block level. Moving it to the package simply won't work that way. I'm ok with the current approach. What's the most important is that it works, so let's go with it.